web/src/pages/roadmap.tsx

```diff
@@ -100,20 +103,33 @@ const RoadmapPage = (props: Props): ReactElement => {
             }
           />
         )}
-        {successAlert && (
+        {profileUpdatedAlert && (
           <ToastAlert
             variant="success"
-            isOpen={successAlert}
+            isOpen={profileUpdatedAlert}
             close={() => {
-              setSuccessAlert(false);
+              setProfileUpdatedAlert(false);
               router.replace({ pathname: "/roadmap" }, undefined, { shallow: true });
             }}
             heading={Config.profileDefaults.successTextHeader}
-            dataTestid="toast-alert-SUCCESS"
           >
             {Config.profileDefaults.successTextBody}
           </ToastAlert>
         )}
+        {calendarAlert && (
```

Comment:
These alerts look pretty duplicative, you could probably refactor them, but not necessary to complete this PR

Reply:
Added a story in the refactoring section of the backlog. We can look at it during refactor fridays.